templates/start-slave.sh.erb

```diff
@@ -1,4 +1,68 @@
 #!/usr/bin/env bash
 
-source <%= @defaults_location %>/jenkins-slave 
-java -jar <%= @slave_home %>/<%= @client_jar %> $JENKINS_SLAVE_ARGS
+source <%= @defaults_location %>/jenkins-slave
+
+# mandatory input vars
```

Comment:
Sounds like a good plan.
I think I did it this way because it wasn't an exact copy of the linux script.
I will see if I can reuse the linux script.